capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- Please, kindly ensure to capitalize the first letter of the given name. use `capitalize` to fix this issue.

Reply:
Hi, @zieeco!
Thanks for the feedback! I have already modified the method by replacing `upcase()` with `capitalize().` 😁